assets/scripts/months.js

```diff
@@ -157,9 +157,21 @@ $(document).ready(function() {
     element.find('td').off('change').off('validate');
 
   	element.find('td').on('change', function (evt) {
+
+
       calculateData();
       makeWaffleChart();
   	}).on('validate', function (evt, value) {
+
+  	  console.log(`Our current event is: ${value}`);
```

Comment:
Without checking for the class of this, the code is running on the .monthsevent as well as .eventname